variables.tf

```
@@ -173,14 +173,26 @@ variable "env_raw" {
   default     = []
 }
 
+variable "pod_security_context" {
+  description = "Pod securityContext"
+  type        = map(string)
```

Should be 

```suggestion
  type        = map(any)
```

Reason being the values of the map can be either string/numbers.